django/applications/catmaid/static/js/widgets/stack-viewer-grid.js

```diff
@@ -621,9 +669,13 @@
   };
 
   StackViewerGrid.prototype.moveStackViewer = function(stackViewer, coords, completionCallback) {
+    var currentCoords = stackViewer.projectCoordinates();
+
     stackViewer.moveToProject(
-      coords.z, coords.y, coords.x,
-      this.sourceStackViewer.primaryStack.stackToProjectSX(this.sourceStackViewer.s),
+      'z' in coords? coords.z : currentCoords.z,
+      'y' in coords? coords.y : currentCoords.y,
+      'x' in coords? coords.x : currentCoords.x,
+      's' in coords ? coords.s : this.sourceStackViewer.primaryStack.stackToProjectSX(this.sourceStackViewer.s),
```

Comment:
Minor nit-picking: would you mind adding spaces in front of the z, y, and x question marks?